app/code/Magento/ReviewGraphQl/Model/Resolver/Product/RatingSummary.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ReviewGraphQl\Model\Resolver\Product;
+
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Catalog\Model\Product;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\ReviewGraphQl\Model\ResourceModel\GetProductRatingSummary;
+
+/**
+ * Product rating summary
+ */
+class RatingSummary implements ResolverInterface
+{
+    /**
+     * @var GetProductRatingSummary
+     */
+    private $getProductRatingSummary;
+
+    /**
+     * RatingSummary constructor.
+     *
+     * @param GetProductRatingSummary $getProductRatingSummary
+     */
+    public function __construct(
+        GetProductRatingSummary $getProductRatingSummary
+    ) {
+        $this->getProductRatingSummary = $getProductRatingSummary;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ): float {
+        /** @var Product $product */
+        $product = $value['model'];
```

Comment:
We need to make sure that the `$value['model']` exists. Otherwise, we may get some unpredictable output.

Reply:
Please, check other places as well

Reply:
Added ✅